lxd/instance/drivers/driver_qemu.go

```diff
@@ -2309,6 +2325,63 @@ func (d *qemu) deviceAttachBlockDevice(deviceName string, configCopy map[string]
 	return nil
 }
 
+func (d *qemu) unmountPath(deviceName string, rawConfig deviceConfig.Device) error {
```

Comment:
In general, does this still allow disk shares to be removed on a stopped VM, even though the clean up cannot occur?

Reply:
I don't think so. I'll have to check but since I can't have access to the agent when the VM is stopped, I won't be able to make use of the `devlxdEventSend` function..

Reply:
fixed